src/__test__/index.test.ts

```diff
@@ -278,15 +278,19 @@ describe('Client', function() {
       existingPerson = entities.pop()!
     })
     it('should stream responses.', done => {
-      client.listen<Person>(store.id, 'Person', existingPerson.ID, reply => {
-        const entity = reply.entity
+      const close = client.listen<Person>(store.id, 'Person', existingPerson.ID, (reply, err) => {
+        const entity = reply?.entity
         expect(entity).to.not.be.undefined
         expect(entity).to.have.property('age')
-        expect(entity.age).to.be.greaterThan(21)
-        events.push(entity.age)
+        expect(entity?.age).to.be.greaterThan(21)
```

Comment:
The only annoying thing with the above is that now, reply is optional... so we have to check for that everywhere 🤷‍♂ .